filament/src/details/Camera.cpp

```diff
@@ -260,7 +275,11 @@ math::details::TMat44<T> inverseProjection(const math::details::TMat44<T>& p) no
 // ------------------------------------------------------------------------------------------------
 
 CameraInfo::CameraInfo(FCamera const& camera) noexcept {
-    projection         = mat4f{ camera.getProjectionMatrix() };
+    projection         = mat4f{ camera.getProjectionMatrix(0) };
+    eyeProjection[0]   = mat4f{ camera.getProjectionMatrix(0) };
```

Comment:
do we need both `projection` and `eyeProjection[0]` ? that's going to be confusing. Maybe we need to bite the bullet and change `projection` everywhere?